docs/setup/docker.asciidoc

```diff
@@ -39,6 +39,13 @@ Alternatively, you can download other Docker images that contain only features
 available under the Apache 2.0 license. To download the images, go to 
 https://www.docker.elastic.co[www.docker.elastic.co].
 
+=== Running Kibana from the Command Line
+==== Development mode
+Kibana can be quickly started and connected to a local Elasticsearch container for development 
+or testing use with the following command:
+--------------------------------------------
+docker run --net=host -e "ELASTICSEARCH_URL=http://localhost:9200" {docker-repo}:{version}
```

Comment:
We should use `ELASTICSEARCH_HOSTS` instead of the old and deprecated one `ELASTICSEARCH_URL`.

Reply:
If the container requires environment variables in order to run it is nice to have an example of them in the getting started section of the doc.  I'll update my branch to use the _HOSTS variable.  I think need to fix the capitalization in the section title as well